src/node_file.cc

```diff
@@ -108,6 +115,7 @@ FSReqWrap* FSReqWrap::New(Environment* env,
   that = new(storage) FSReqWrap(env, req, syscall, data);
   if (copy)
     that->data_ = static_cast<char*>(memcpy(that->inline_data(), data, size));
+  that->encoding_ = encoding;
```

Comment:
If possible, try assigning it in the constructor so you can make the field `const`.